src/esw/brushed_motors/Controller.cpp

```diff
@@ -197,12 +198,45 @@ void Controller::turnOn() const {
 }
 
 // REQUIRES: nothing
-// MODIFIES: isLive
+// MODIFIES: nothing
+// EFFECTS: Returns a combined ID for both the deviceAddress and motorID
+uint8_t Controller::combineDeviceMotorID() const {
+    return (deviceAddress << 3) | motorID;
```

Comment:
Oh we should add a comment for why we are doing this (motorID is known to be 3 bits long and device address is currently only two bits long, 1 or 2). Also, on reflection, it might be useful to have an assert inside the Controller constructor making sure that the motorID is never greater than 3 bits long (you can do a (motorID & 0b111) == motorID).